recipes/batchspawner/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: batchspawner
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/b/batchspawner/batchspawner-{{ version }}.tar.gz
+  sha256: 9bae72f7c1bd9bb11aa58ecc3bc9fae5475a10fdd92dc0c0d67fa7eb95c9dd3a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.3
+  run:
+    - __unix
+    - async_generator >=1.8
+    - jinja2
+    - jupyterhub-base >=0.5
+    - python >=3.3
+
+test:
+  imports:
+    - batchspawner
+  requires:
+    - pip
+    - packaging  # TODO: remove after https://github.com/conda-forge/jupyterhub-feedstock/issues/59
+  commands:
+    - pip check
+    # even this expects a running hub
+    # - batchspawner-singleuser --version
```

Comment:
Maybe could do `command -v batchspawner-singleuser` or check for this in `$PREFIX/bin` along with something similar on Windows

Reply:
> something similar on Windows

_luckily_ not a concern. I intend to try to get a `--version` added upstream, and use `console_scripts`.